weaviate/collections/classes/filters.py

```diff
@@ -96,7 +94,7 @@ class _GeoCoordinateFilter(GeoCoordinate):
 
 
 FilterValuesList = Union[
-    List[str], List[bool], List[int], List[float], List[datetime], List[uuid_lib.UUID]
+    List[str], List[bool], List[int], List[float], List[datetime], Sequence[UUID]
```

Comment:
should we migrate all Lists to Sequence here?

Reply:
I had thought not but upon thinking about it would be better if we did, so will add!